plugin/wavesurfer.regions.js

```diff
@@ -41,7 +41,7 @@ WaveSurfer.Regions = {
         var pxMove = 0;
 
         var eventDown = function (e) {
-            if (e.touches && e.touches.length > 1) { return; }
+            if (e.touches && e.touches.length > 1 || e.target.childElementCount > 0) { return; }
```

Comment:
Thanks for the solution!

I think it should go a separate condition for consistency across the event callbacks:

```
if (e.touches && e.touches.length > 1) { return; }
// Check whether the click/tap is on the bottom-most DOM-element
if (e.target.childElementCount > 0) { return; }
```

Please also don't add the minified files from `dist` in your commit. I'll build & commit it before an NPM release.